README.md

```diff
@@ -148,10 +149,10 @@ It formats the exported .rtf file into the format required by lots of Elsevier J
 
 ### Kate Turabian’s A Manual for Writers of Term Papers, Theses, and Dissertations
 
-Documentation is at [Chicago (English)/README.md](Chicago (English)/README.md).
-
-- Directory: [Turabian (English)](Turabian (English)/) and [Turabian (Spanish)](Turabian (Spanish)/)
+Documentation is at [Turabian (English)//TURABIAN.README](Turabian%20(English)//TURABIAN.README).
```

Comment:
I wonder about the two slashes here. Should be only one in URLs. 🤔

Reply:
... yes, this is wrong... And the referenced files is not existing at all... I wonder where the hell I copied that file name?!